src/Microsoft.AspNetCore.Razor.Language/ClassifiedSpanVisitor.cs

```diff
@@ -0,0 +1,336 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Razor.Language.Legacy;
+using Microsoft.AspNetCore.Razor.Language.Syntax;
+
+namespace Microsoft.AspNetCore.Razor.Language
+{
+    internal class ClassifiedSpanVisitor : SyntaxRewriter
```

Comment:
>ClassifiedSpanVisitor [](start = 19, length = 21)

At this point I've gotten through all of the files with a small number of changes or those that were copied from Roslyn. My remaining list to review:

ClassifiedSpanVisitor
DefaultRazorIntermediateNodeLoweringPhase
DefaultRazorTagHelperBinderPhase
CSharpCodeParser
HtmlMarkupParser
TagHelperParseTreeRewriter
TokenizerBackedParser
WhitespaceRewriter
MarkupElementRewriter